docs(repo): Link to package README files (#184)
BfArM-MVH/grz-tools: README.md

# GRZ Tools Monorepo

This monorepo hosts the following packages:

> - [`grz-cli`](packages/grz-cli/README.md) - A command-line tool for validating, encrypting, uploading and downloading submissions to/from a GDC/GRZ (Genomrechenzentrum).

Reviewer: I would not mention downloading for `grz-cli`, since it should only be uploading. And only to a GDC/GRZ, not from (that's grzctl)

> - [`grz-pydantic-models`](packages/grz-pydantic-models/README.md) - Pydantic models of the BfArM GRZ metadata schema.

Reviewer: ```suggestion
- [`grz-pydantic-models`](packages/grz-pydantic-models/README.md) - Pydantic models for schemas related to the genomDE Model Project.
```

- [`grzctl`](packages/grzctl/README.md) - GRZ internal tooling.
- [`grz-common`](packages/grz-common/README.md) - Common code shared between packages in `grz-tools`.

> - [`grz-db`](packages/grz-db/README.md) - Libraries, SQL models and alembic migrations for GRZ DB.

Reviewer: ```suggestion
- [`grz-db`](packages/grz-db/README.md) - Libraries, SQL models and alembic migrations for the GRZ internal submission DB.
```

## grz-cli

The [`grz-cli`](packages/grz-cli/README.md) package is the primary CLI for submissions to the GRZs.
It provides functionality for:
- Validating submissions
- Encrypting files using crypt4gh

> - Uploading files to/from a GRZ

Reviewer: ```suggestion
- Uploading files to a GRZ
```
